Formula/rpki-client.rb

```diff
@@ -0,0 +1,49 @@
+class RpkiClient < Formula
+  desc "OpenBSD portable rpki-client"
+  homepage "https://github.com/rpki-client/rpki-client-portable"
+  url "https://github.com/rpki-client/rpki-client-portable/archive/refs/tags/7.4.tar.gz"
+  sha256 "cacb56107b29baef67c0117bc53d726c95684ad27a6affe2a1527470c67f5e86"
+  license "ISC"
+
+  depends_on arch: :intel
+  depends_on "autoconf"
+  depends_on "automake"
+  depends_on "libressl"
+  depends_on "libtool"
```

Comment:
Is this needed at runtime?

Reply:
libressl is required at runtime, the others are not, and I will adopt the same ```=> :build``` model as above